mypy.ini

```diff
@@ -61,3 +61,6 @@ ignore_missing_imports = True
 
 [mypy-mordred]
 ignore_missing_imports = True
+
+[mypy-polars]
+ignore_missing_imports = True
```

Comment:
does polars not have types or stubs?

Reply:
Yes, I faced `no stubs found` error and added this line.

Reply:
im aksing becuase sometimes there are dedicated stubs packages that need to be isntalled, but if you checked and theres nothing please resolve

Reply:
I couldn't find stub package for polars